ansible/wireguard_sn3.yaml

```diff
@@ -135,6 +135,9 @@ wireguard_configs:
     NEIGHBORS: "10.70.250.178"
     TX_LENGTH: 1420
     COST: 30
+    BFD_ENABLE: yes
```

Comment:
Can we try this in a new config on [dev](https://github.com/nycmeshnet/vpn-infra/blob/main/ansible/dev.yaml) before pushing it to prod?

Reply:
Note it is a different IP range. You can leave this one here, it'll just only be deployed when this is merged.

Reply:
@james-otten didnt realize we had a dev instance to test. Happy to do that. Will action tomorrow

Reply:
Thanks!

Reply:
Yes please. The same format can be used in `dev.yaml` it is just a different IP range

Reply:
Did the thing: https://github.com/nycmeshnet/vpn-infra/actions/runs/14346005585/job/40215835977 note the IP change

Reply:
Good catch https://github.com/nycmeshnet/vpn-infra/actions/runs/14346044229